devnet-sdk/book/src/testing.md

```diff
@@ -85,16 +85,16 @@ func wrapETHScenario(chainIdx uint64, walletGetter validators.WalletGetter) syst
         user := walletGetter(ctx)
         
         // Access contract registry
-        wethAddr := constants.SuperchainWETH
-        weth, err := chain.ContractsRegistry().SuperchainWETH(wethAddr)
+        bridgeAddr := constants.SuperchainETHBridge
+        weth, err := chain.ContractsRegistry().SuperchainETHBridge(bridgeAddr)
```

Comment:
I don't think this is right?  [Looks like the ContractsRegistry](https://github.com/ethereum-optimism/optimism/blob/a979a9444dbb482843f2a42f437ced54a8ac1053/devnet-sdk/interfaces/registry.go#L20-L24) only has a WETH contract, so I think this should be:
```suggestion
        wethAddr := constants.WETH
        weth, err := chain.ContractsRegistry().WETH(bridgeAddr)
```